src/Uno.UI/UI/Xaml/Markup/Reader/XamlObjectBuilder.cs

```diff
@@ -159,6 +159,13 @@ public XamlObjectBuilder(XamlFileDefinition xamlFileDefinition)
 				var instance = component ?? Activator.CreateInstance(type)!;
 				rootInstance ??= instance;
 
+				var instanceAsFrameworkElement = instance as FrameworkElement;
+
+				if (instanceAsFrameworkElement is not null)
```

Comment:
```suggestion
				if (instance is FrameworkElement instanceAsFrameworkElement)
```

Reply:
This one is not possible because of the later use. `instanceAsFrameworkElement would be an uninitialized variable.